tests/conftest.py

```diff
@@ -20,3 +24,10 @@ def remove_data_folder():
             shutil.rmtree(data)
 
     request.addfinalizer(remove_data_folder)
+
+
+@pytest.fixture
+def tas_series():
+    """Return mean temperature time series."""
+    _tas_series = partial(test_timeseries, variable="tas")
+    return _tas_series
```

Comment:
This is the xclim solution, where we avoided a huge PR by keeping the same fixture. However, I'm not sure I see any gain in using fixtures here ?

I would have suggested importing `test_timeseries` in each test file and using the function directly in the tests. This will  make for a simpler `conftest.py`.

Reply:
```
import numpy as np
import xscen as xs
from xclim.testing.helpers import test_timeseries
import pytest


class TestClimatologicalMean:
    def test_daily(self):
        ds = test_timeseries(np.tile(np.arange(1, 13), 3), variable="tas", start="2001-01-01", freq="D", as_dataset=True)
        with pytest.raises(NotImplementedError):
            xs.climatological_mean(ds)
```
I get this error when trying to use it directly:
```
___________________________________________________________________________________________________________________ ERROR at setup of test_timeseries ____________________________________________________________________________________________________________________
  def test_timeseries(
E       fixture 'values' not found
>       available fixtures: anyio_backend, anyio_backend_name, anyio_backend_options, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cleanup_notebook_data_folder, cov, doctest_namespace, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tas_series, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, xdoctest_namespace
>       use 'pytest --fixtures [testpath]' for help on them.

```

Reply:
Je'l savais merde.

```
from xclim.testing.helpers import test_timeseries as timeseries
```

When a member function of a `test.py` file starts with `test_` it is considered a test and pytest tries to run it so. `values` is the first arg of the function and pytest assumes it's a fixture.